src/main/resources/static/css/styles.css

```diff
@@ -48,3 +55,38 @@ body {
   background: -webkit-linear-gradient(left, #fff 0, #6ecbd4 15%, #6ecbd4 85%, #fff 100%);
   background: linear-gradient(to right, #fff 0, #6ecbd4 15%, #6ecbd4 85%, #fff 100%)
 }
+
+.pull-right { float: right; }
+.pull-left { float: left; }
+
+.product-grid {
+  padding: 0;
+  width: 420px;
+  margin: 10px auto;
+  list-style: none;
+}
+
+.product-grid .item {
+  float: left;
+  display: block;
+  width: 180px;
+  margin: 0 5px 20px;
+  border: 1px solid rgb(249, 249, 249);
+  box-shadow: 0 1px 0 rgba(0, 0, 0, 0.2);
+}
+
+.product-grid .item img {
```

Comment:
pleasee don't style html tags. Provide a className, e.g. `product-image` and then style it.